src/addDelegator.ts

```diff
@@ -59,7 +59,7 @@ export async function getUnsignedAddDelegator(
   }
 }
 
-async function getAddDelegatorParams(
+export async function getAddDelegatorParams(
```

Comment:
since the function is export please add function description in comment as documentation .